modules/modelSaver/StableDiffusionLoRAModelSaver.py

```diff
@@ -26,7 +26,8 @@ def save(
         embedding_model_saver = StableDiffusionEmbeddingSaver()
 
         lora_model_saver.save(model, output_model_format, output_model_destination, dtype)
-        embedding_model_saver.save_multiple(model, output_model_format, output_model_destination, dtype)
+        if not model.train_config.bundle_additional_embeddings:
```

Comment:
You only made this change here and in `StableDiffusionXLLoRAModelSaver`. Is it not needed in `PixArtAlphaLoRAModelSaver` and `WuerstchenLoRAModelSaver`?

Reply:
Oh, nice catch. Totally forgot about those.